extensions/wikia/Wall/js/WallBackendBridge.js

```diff
@@ -155,7 +155,8 @@
 					isreply: isreply,
 					pagetitle: page.title,
 					pagenamespace: page.namespace,
-					convertToFormat: convertToFormat
+					convertToFormat: convertToFormat,
+					token: window.mw.user.tokens.get('editToken')
 				},
 				callback: this.proxy(function (data) {
```

Comment:
@kvas-damian, this extension JS has no error handling.  As an anonymous user, try clicking 'edit' next to an article comment (or Forum after logging out in another tab) and you'll see these errors just get eaten.

I don't think we care enough about this feature to add this new functionality, but if we do its another ticket.